forestserviceprototype/specialuseform/models.py

```diff
@@ -9,25 +9,40 @@ class Permit(models.Model):
         ('needs_approval', 'Needs Approval'),
         ('approved', 'Approved'),
         ('in_review', 'In Review'),
-        ('not_approved', 'Rejected')
```

Comment:
This was already covered in #17 (https://github.com/18F/forest-service-prototype/blob/577a0ddc1186dfc684c6a9a090891079943e5c0d/forestserviceprototype/specialuseform/models.py), no?